spread.yaml

```
@@ -73,6 +73,17 @@ restore-each: |
   lxc --project=rockcraft stop --all
   lxc --project=rockcraft delete $(lxc --project=rockcraft list -c n --format csv)
 
+debug-each: |
+  # output latest rockcraft log file on test failure
+  rockcraft_log_file=$(find /root/.cache/rockcraft/log/ -name 'rockcraft*.log' | sort -n | tail -n1)
+  if [[ -f $rockcraft_log_file ]]; then
+    echo "rockcraft log file contents ($rockcraft_log_file)"
```

```suggestion
    echo ">>>>>> rockcraft log file contents ($rockcraft_log_file)"
```
(just to make it easier to find this in the sea of output)

---

Good idea, let me try some line breaks